ngraph_bridge/ngraph_encapsulate_impl.cc

```diff
@@ -558,13 +558,20 @@ Status NGraphEncapsulateImpl::UpdatePipelinedTensorCache(
     // Create these pipelined ng tensors only if needed, else reuse from cache
```

Comment:
Need to remove this
https://github.com/tensorflow/ngraph-bridge/pull/356/files#diff-a499ec3819d5249725dc073c6d8d7af7R11
and this https://github.com/tensorflow/ngraph-bridge/pull/356/files#diff-984639fceaea8131336f5fd39edd6450R165

we were disabling NoOp to get by the error. Will look into rest of the PR later

Reply:
Are there any unit tests that can be added?

Reply:
maybe a unit test is we create a PTS with 2 PipelinedTensorMatrix, with different depths. The construction should throw an error

Reply:
Added the simple test